### ISSUE TITLE
Router.current() returns null

### ISSUE THREAD

**Issue by dohomi** [zimme/meteor-active-route#13]
Hello,

I tried your package in current iron:router but Router.current() returns null. I make use of this package and ```martino:iron-router-i18n``` package. 

If I extend package.js:

```
  api.use([
    'iron:router@1.0.0',
    'martino:iron-router-i18n',
    'underscorestring:underscore.string@2.4.0'
  ]);
```
Then Router.current() works as expected. Maybe @martinopic has also an idea about it? Thanks

**Comment by martinopic** [comment]
Have you tried to load 'iron-router-i18n' before meteor-iron-router-active ?. Iron Router i18n has to replace the Router originally created by Iron Router with a new one and this might cause some problems (still working to see if there is a safer way to do this).

**Comment by zimme** [comment]
First thought was you need to guard `Router.current()` as it's possible for it to be null, i.e. when no route has run yet. (It's in iron:router docs)

Second thought was, I don't think it's a problem with this package but with `martino:iron-router-i18n` as no other users has brought this issue up with `iron:router@1.0`.

If it turns out I'm wrong, ping me and I'll take a look at my package to see if there's anything I can do.

**Comment by martinopic** [comment]
Ok, this ended up to be a problem with Iron Router i18n. I've fixed that, see the above referenced ticket for details.

Now the two module works iff ```martino:iron-router-i18n``` is added (loaded) BEFORE ```zimme:iron-router-active```. I've also added a basic example app working with both.

This is needed because I am overriding and replacing the original iron router I'm also discussing in a more general way how to do it in the easiest and painless possible way here: https://github.com/yoolab/iron-router-i18n/issues/32
